01-backward.go

```diff
@@ -1,62 +1,22 @@
 package main
 
-// #cgo CFLAGS: -I ./cgotorch
-// #cgo LDFLAGS: -L ./cgotorch -Wl,-rpath ./cgotorch -lcgotorch
-// #cgo LDFLAGS: -L ./cgotorch/libtorch/lib -Wl,-rpath ./cgotorch/libtorch/lib -lc10 -ltorch -ltorch_cpu
-// #include "cgotorch.h"
-import "C"
-import "fmt"
+import (
+	"fmt"
 
-type Tensor struct {
-	T C.Tensor
-}
-
-func RandN(rows, cols int, require_grad bool) Tensor {
-	rg := 0
-	if require_grad {
-		rg = 1
-	}
-	return Tensor{C.RandN(C.int(rows), C.int(cols), C.int(rg))}
-}
-
-func MM(a, b Tensor) Tensor {
-	return Tensor{C.MM(a.T, b.T)}
-}
-
-func Sum(a Tensor) Tensor {
-	return Tensor{C.Sum(a.T)}
-}
-
-func (a Tensor) String() string {
-	s := C.Tensor_String(a.T)
-	r := C.GoString(s)
-	C.FreeString(s)
-	return r
-}
-
-func (a Tensor) Print() {
-	C.Tensor_Print(a.T)
-}
-
-func (a Tensor) Backward() {
-	C.Tensor_Backward(a.T)
-}
-
-func (a Tensor) Grad() Tensor {
-	return Tensor{C.Tensor_Grad(a.T)}
-}
+	"github.com/wangkuiyi/gotorch/at"
+)
 
 func main() {
-	a := RandN(3, 4, true)
+	a := at.RandN(3, 4, true)
```

Comment:
Do we have to reuse the name *at*? *at* is the abbreviation of *ATen*, which is itself the abbreviation of ***A TEN**sor library for C++11*

Reply:
As the comment https://github.com/wangkuiyi/gotorch/pull/9#pullrequestreview-455729294, I would resue the namespace of torch.